tests/functional/services/policy_engine/test_feed_sync.py

```diff
@@ -0,0 +1,104 @@
+import os
+
+import tests.functional.services.policy_engine.utils.api as policy_engine_api
+from tests.functional.services.policy_engine.conftest import FEEDS_DATA_PATH_PREFIX
+from tests.functional.services.utils import http_utils
+
+# TODO check schema
+
+
+class TestFeedSync:
+    @classmethod
+    def _find_by_name(cls, records, name):
+        """
+        From a list of objects/dictionaries, selects first index with matching name. Returns None if nothing is found
+        :param records: list of objects or dictionaries that are expected to have 'name' attr
+        :type records: list
+        :return: dict with matching name or 'None' if nothing found
+        :rtype: Union[dict, None]
+        """
+        for record in records:
+            if record["name"] == name:
+                return record
+        return None
+
+    @classmethod
+    def _get_vuln_ids(cls, expected_vulns):
+        """
+        From a list of expected vulns taken from feeds service, find corresponding vulnerability ids for querying anchore
+        :param expected_vulns: list of records in group from feed
+        :type expected_vulns: list
+        :return: list of vulnerabilitiy_ids as they would be stored in anchore
+        :rtype: list
+        """
+        vuln_ids = []
+        for vuln in expected_vulns:
+            if "Advisory" in vuln:
+                if "ghsaId" in vuln["Advisory"]:
+                    vuln_ids.append(vuln["Advisory"]["ghsaId"])
+                else:
+                    vuln_ids += vuln["Advisory"]["CVE"]
+            if "@id" in vuln:
+                vuln_ids.append(vuln["@id"])
+            if "cve" in vuln:
+                vuln_ids.append(vuln["cve"]["CVE_data_meta"]["ID"])
```

Comment:
Similar to above above, will "cve" always have ["CVE_data_meta"]["ID"] nested under it?

Reply:
@dspalmer99 Similar answer to the other question. This condition is for the nvdv2 feed and with our static set of feed data this has not presented any issues. It would only ever become a problem if someone introduces new data that does not meet this criteria

Reply:
Correct. If there was a new feed introduced that had the "cve" key in it then this would start to fail. To my knowledge, there are no other feeds that are currently structured this way.

Reply:
We can remove the @id check, as that was for nvd (not v2).

Reply:
@id key case removed